tools/dart-api-builder/index.js

```diff
@@ -0,0 +1,93 @@
+var fs = require('fs');
+var path = require('canonical-path');
+var Package = require('dgeni').Package;
+var basePackage = require('../api-builder/docs-package');
+var targetPackage = require('../api-builder/target-package');
+var cheatsheetPackage = require('../api-builder/cheatsheet-package');
+
+var PROJECT_PATH = path.resolve(__dirname, "../..");
+var PUBLIC_PATH = path.resolve(PROJECT_PATH, 'public');
+var DOCS_PATH = path.resolve(PUBLIC_PATH, 'docs');
+var ANGULAR_REPO_PATH = path.resolve(__dirname, '../../../angular-dart');
+var ANGULAR2_DOCS_PATH = path.resolve(ANGULAR_REPO_PATH, 'docs');
+var NG_IO_PKG_PATH = path.resolve(__dirname, "../api-builder/angular.io-package");
+
+function requireNgIoPkg(_path) { return require(path.resolve(NG_IO_PKG_PATH, _path)); }
+
+module.exports = new Package('dart-api-and-cheatsheet-builder', [basePackage, targetPackage, cheatsheetPackage])
+
+  // overrides base packageInfo and returns the one for the Angular repo.
+  .factory(require('./services/packageInfo'))
+
+  // Configure rendering
+  .config(function (templateFinder, renderDocsProcessor) {
+
+    templateFinder.templateFolders
+      .unshift(path.resolve(NG_IO_PKG_PATH, 'templates'));
+
+    // helpers are made available to the nunjucks templates
+    renderDocsProcessor.helpers.relativePath = function (from, to) {
+      return path.relative(from, to);
+    };
+  })
+
+  .config(function (parseTagsProcessor, getInjectables) {
+    const tagDefs = requireNgIoPkg('./tag-defs');
+    parseTagsProcessor.tagDefinitions =
+      parseTagsProcessor.tagDefinitions.concat(getInjectables(tagDefs));
+  })
+
+  .config(function (readTypeScriptModules, writeFilesProcessor, readFilesProcessor) {
```

Comment:
No, we only need to read read `'cheatsheet/*.md'` files as input; see line 49 below.